passacre/config.py

```diff
@@ -303,10 +307,37 @@ def set_config(self, site, name, value):
                 (site, name, jdumps(new_value)))
         self._db.commit()
 
+class SqliteTahoeConfig(SqliteConfig):
+    @features.tahoe.check
+    def __init__(self):
+        try:
+            with open("~/.tahoe/node.url") as f:
+                self.tahoe = f.readline().strip()
+                self.tahoe += "uri/" if self.tahoe.endswith("/") else "/uri/"
```

Comment:
`urllib.parse.urljoin` kinda sucks rocks, so i'm not changing this. sorry-not-sorry.